doc/cephfs/mds-config-ref.rst

```diff
@@ -15,420 +15,50 @@
-
-:Description: Ceph will inject MDS failure in the subtree export code 
-              (for developers only).
-              
-:Type:  32-bit Integer
-:Default: ``0``
-
-
-``mds_kill_import_at``
-
-:Description: Ceph will inject MDS failure in the subtree import code 
-              (for developers only).
-              
-:Type:  32-bit Integer
-:Default: ``0``
-
-
-``mds_kill_link_at``
-
-:Description: Ceph will inject MDS failure in hard link code 
-              (for developers only).
-              
-:Type:  32-bit Integer
-:Default: ``0``
-
-
-``mds_kill_rename_at``
-
-:Description: Ceph will inject MDS failure in the rename code 
-              (for developers only).
-              
-:Type:  32-bit Integer
-:Default: ``0``
-
-
-``mds_wipe_sessions``
-
-:Description: Ceph will delete all client sessions on startup 
-              (for testing only).
-              
-:Type:  Boolean
-:Default: ``false``
-
-
-``mds_wipe_ino_prealloc``
-
-:Description: Ceph will delete ino preallocation metadata on startup 
-              (for testing only).
-              
-:Type:  Boolean
-:Default: ``false``
-
-
-``mds_skip_ino``
-
-:Description: The number of inode numbers to skip on startup 
-              (for testing only).
-              
-:Type:  32-bit Integer
-:Default: ``0``
-
-
-``mds_min_caps_per_client``
-
-:Description: Set the minimum number of capabilities a client may hold.
-:Type: Integer
-:Default: ``100``
-
-
-``mds_max_ratio_caps_per_client``
-
-:Description: Set the maximum ratio of current caps that may be recalled during MDS cache pressure.
-:Type: Float
-:Default: ``0.8``
+.. confval:: mds_early_reply
+.. confval:: mds_default_dir_hash
+.. confval:: mds_log_skip_corrupt_events
+.. confval:: mds_log_max_events
+.. confval:: mds_log_max_segments
+.. confval:: mds_bal_sample_interval
+.. confval:: mds_bal_replicate_threshold
+.. confval:: mds_bal_unreplicate_threshold
+.. confval:: mds_bal_split_size
+.. confval:: mds_bal_split_rd
+.. confval:: mds_bal_split_wr
+.. confval:: mds_bal_split_bits
+.. confval:: mds_bal_merge_size
+.. confval:: mds_bal_interval
+.. confval:: mds_bal_fragment_interval
+.. confval:: mds_bal_fragment_fast_factor
+.. confval:: mds_bal_fragment_size_max
+.. confval:: mds_bal_idle_threshold
+.. confval:: mds_bal_max
+.. confval:: mds_bal_max_until
+.. confval:: mds_bal_mode
+.. confval:: mds_bal_min_rebalance
+.. confval:: mds_bal_min_start
+.. confval:: mds_bal_need_min
+.. confval:: mds_bal_need_max
+.. confval:: mds_bal_midchunk
+.. confval:: mds_bal_minchunk
+.. confval:: mds_replay_interval
+.. confval:: mds_shutdown_check
+.. confval:: mds_thrash_exports
+.. confval:: mds_thrash_fragments
+.. confval:: mds_dump_cache_on_map
+.. confval:: mds_dump_cache_after_rejoin
+.. confval:: mds_verify_scatter
+.. confval:: mds_debug_scatterstat
+.. confval:: mds_debug_frag
+.. confval:: mds_debug_auth_pins
+.. confval:: mds_debug_subtrees
+.. confval:: mds_kill_mdstable_at
+.. confval:: mds_kill_export_at
```

Comment:
Why do some of these options not show the default value and other do? e.g. `mds_bal_min_rebalance` does show the default.

Reply:
@batrick hi Patrick, thank you for your review. indeed, it's a bug in the sphinx extension. fixed and repushed. could you please take another look?

Reply:
Looks great, thanks @tchaikov